src/yetibot/core/commands/karma.clj

```diff
@@ -11,11 +11,11 @@
 
 (def config (:value (get-config sch/Any [:karma])))
 
-(def pos-emoji (or (-> config :emoji :positive) "🌈"))
-(def neg-emoji (or (-> config :emoji :negative) "⛈"))
+(def pos-emoji (or (-> config :emoji :positive) ":rainbow:"))  ;; 🌈
```

Comment:
Should we add a comment noting our decision to support Slack only for these with the possible future goal of supporting both with a shortcode-to-emoji mapping?

Reply:
Sure.  I will add a note to the top of this command's namespace.